### ISSUE TITLE
Homework Week 6

### ISSUE THREAD

**Issue by ptums** [JacobDeming/GIF-Search#1]
Nice job on the homework! You nailed all the features? Were you able to refactor it?

**Comment by JacobDeming** [comment]
Yep! Everything in there should work properly so long as it has not broken recently.
